pkg/api/meta/restmapper.go

```diff
@@ -335,3 +359,23 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
+		if t.ResourceInGroup(group, resource) {
+			return true
+		}
+	}
+	return false
+}
+
+// IsResource takes a string and checks if it's a resource
```

Comment:
In this case this method returns true (this resource is valid), which I believe is what we expected?